src/utils/events/move-input.js

```diff
@@ -24,9 +26,29 @@ export default class MoveInput {
   }
 
   handler(event) {
```

Comment:
> So, while it's not idiomatic in our frameworks, it is so for Hammer...

Thanks, that explains it, I should have realized this. Still, maintainers and reviewers might not have objected to a short comment indicating this.